elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -95,6 +95,8 @@ class PfeAccordion extends PFElement {
     this.addEventListener(`${PfeAccordion.tag}:change`, this._changeHandler);
     this.addEventListener("keydown", this._keydownHandler);
 
+    let numberItems = this.querySelectorAll('pfe-accordion > pfe-accordion-header');
```

Comment:
Is this selector working at the moment?  I think the this context _is_ the pfe-accordion so perhaps it should be: `this.querySelectorAll(":scope > pfe-accordion-header")`?

Reply:
Although it doesn't appear this is being used below so maybe we delete it?

Reply:
Shoot. Yep, meant to delete it.